Src/FluentAssertions/Collections/GenericCollectionAssertions.cs

```diff
@@ -810,13 +810,18 @@ public AndConstraint<TAssertions> Contain(IEnumerable<T> expected, string becaus
     }
 
     /// <summary>
-    /// Asserts that a collection of objects contains at least one object equivalent to another object.
+    /// Asserts that at least one object in the collection is equivalent to <paramref name="expectation"/>.
+    /// <para>
```

Comment:
`<para>` doesn't work in the summary (and that is expected). Two `<br/>`s will work, but I think moving it to the remarks section is better.

![image](https://github.com/fluentassertions/fluentassertions/assets/572734/33e9a41f-a46a-4f38-824d-83c92a71813f)

Reply:
What does @jnyrup think? 

As I recall, `<remarks>` were not shown in VisualStudio.

Reply:
That's a ~Visual Studio~ Rider limitation. This is what I see in ~Rider~ Visual Studio: 😉 
![image](https://github.com/fluentassertions/fluentassertions/assets/919634/fccc7ebe-7ae5-4bd6-abc9-9fdfd944b15e)

> The `<para>` tag is for use inside a tag, such as `<summary>`, `<remarks>`, or `<returns>` [...]

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#format-documentation-output

On a more serious note, the change of the summary to `Asserts that at least one object in the collection is equivalent to <paramref name="expectation"/>` is enough for me and improves the current situation for both Rider and VS users.
I would only change it from using "object" to "element".

I'm fine having the "you cannot..." part in `<remarks>` - that also seems to be the semantically more correct place to put it.

Reply:
Ah, if it works in VS, then keep it there. I'll report an issue for Rider.

Reply:
To be fair, this is what VS shows in the IntelliSense, but I guess (to their favor) that's a conscious choice to clutter the UI less.

![image](https://github.com/fluentassertions/fluentassertions/assets/919634/da256277-736b-4d61-8b6f-f3df85f61c3e)